pages/agenda/2018.tsx

```diff
@@ -0,0 +1,280 @@
+import * as React from 'react'
```

Comment:
Adding esModuleInterop to tsconfig means we could import `React, {Fragment} from 'react'` and it saves confusion (at least I find the import * confusing)

Reply:
Sounds good, feel free to submit a PR for that! :)

Reply:
Opened #144